crates/ruff_python_ast/src/generated.rs

```diff
@@ -7123,3 +7138,847 @@ pub struct ExprIpyEscapeCommand {
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtClassDef {
+            decorator_list,
+            name,
+            type_params,
+            arguments,
+            body,
+            range: _,
+        } = self;
+
+        for elm in decorator_list {
+            visitor.visit_decorator(elm);
+        }
+        visitor.visit_identifier(name);
+
+        if let Some(type_params) = type_params {
+            visitor.visit_type_params(type_params);
+        }
+
+        if let Some(arguments) = arguments {
+            visitor.visit_arguments(arguments);
+        }
+
+        visitor.visit_body(body);
+    }
+}
+
+impl StmtReturn {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtReturn { value, range: _ } = self;
+
+        if let Some(value) = value {
+            visitor.visit_expr(value);
+        }
+    }
+}
+
+impl StmtDelete {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtDelete { targets, range: _ } = self;
+
+        for elm in targets {
+            visitor.visit_expr(elm);
+        }
+    }
+}
+
+impl StmtTypeAlias {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtTypeAlias {
+            name,
+            type_params,
+            value,
+            range: _,
+        } = self;
+        visitor.visit_expr(name);
+
+        if let Some(type_params) = type_params {
+            visitor.visit_type_params(type_params);
+        }
+
+        visitor.visit_expr(value);
+    }
+}
+
+impl StmtAssign {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtAssign {
+            targets,
+            value,
+            range: _,
+        } = self;
+
+        for elm in targets {
+            visitor.visit_expr(elm);
+        }
+        visitor.visit_expr(value);
+    }
+}
+
+impl StmtAugAssign {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtAugAssign {
+            target,
+            op,
+            value,
+            range: _,
+        } = self;
+        visitor.visit_expr(target);
+        visitor.visit_operator(op);
+        visitor.visit_expr(value);
+    }
+}
+
+impl StmtAnnAssign {
+    pub(crate) fn visit_source_order<'a, V>(&'a self, visitor: &mut V)
+    where
+        V: SourceOrderVisitor<'a> + ?Sized,
+    {
+        let StmtAnnAssign {
+            target,
+            annotation,
+            value,
+            simple: _,
+            range: _,
+        } = self;
+        visitor.visit_expr(target);
+        visitor.visit_expr(annotation);
```

Comment:
I think this should be `visit_annotation`

Reply:
Thanks, corrected.